test/e2e/prometheus_test.go

```diff
@@ -44,6 +44,7 @@ var antreaAgentMetrics = []string{
 	"antrea_agent_ovs_flow_ops_error_count",
 	"antrea_agent_ovs_flow_ops_latency_milliseconds",
 	"antrea_agent_ovs_total_flow_count",
+	"antrea_agent_ovs_ovs_meter_packet_count",
```

Comment:
it this correct? two "ovs"?

Reply:
Nice catch, thanks. It seems that prometheus e2e test is disabled by default on CI. I will double check this one and update.